### ISSUE TITLE
Thanks for this lib ❤️

### ISSUE THREAD

**Issue by martinezguillaume** [carvajalconsultants/apollo-link-offline#9]
Just a quick thanks for this awesome lib !
Works perfectly and makes life easier

**Comment by miguelocarvajal** [comment]
Thanks for the shout out!

Glad to hear you’re having success using it!

May I ask what the use case is?

**Comment by SergeiMeza** [comment]
Thanks for the library. In my case, we will use it for a school web application.

**Comment by Paolo-Gambardella** [comment]
You guys literally saved my project !
Glad to know that @martinezguillaume validate the lib haha

**Comment by miguelocarvajal** [comment]
Glad to hear the library is working well for you guys!

**Comment by planetClaire** [comment]
Just want to add my thanks for saving my project too!  A couple of notes that may help others:

1. Any reason why you haven't been added to apollo's community links? This link has been an absolute lifesaver for me but it took a while to find it...
2. I had to add @client to the syncStatusQuery items so it would play nice with my graphql server - not sure if there's another way around this other than updating your link
2. Speaking of the syncStatusQuery, I like it so much I've added the queue to it so I can display the details of the queue to the user
3. My server was generating primary keys & I was having a hell of a time trying to synchronise results on return from the server until I realised its perfectly fine to generate the primary keys on the client with uuid(). Luckily I had the freedom to be able to change my graphql server
4. Also, a server design issue - I was getting primary key collisions on queued mutations being executed due to retries crossing eachother, my graphql server now implements idempotency, problems gone.
